upgrade/UPGRADE-v9.2.0-dev.md

```diff
@@ -8,6 +8,22 @@ There you can find links to upgrade notes for other versions too.
 ## Composer dependencies
 - remove `vasek-purchart/console-errors-bundle` dependency ([#2408](https://github.com/shopsys/shopsys/pull/2408))
     - see #project-base-diff to update your project
+- upgrade your dependencies to the versions that support both PHP 7.4 and 8.1 ([#2417](https://github.com/shopsys/shopsys/pull/2417))
```

Comment:
does it make sense to write it less generic (in PR description and title too). 
I think it's about doctrine related packages, what do you think?

Reply:
indeed - d20ea16f7 :slightly_smiling_face: